src/monitor/monitor.c

```diff
@@ -67,6 +66,11 @@
  */
 #define KRB5_RCACHE_DIR_DISABLE "__LIBKRB5_DEFAULTS__"
 
+/* for detecting if NSCD is running */
+#ifndef NSCD_SOCKET_PATH
+#define NSCD_SOCKET_PATH "/var/run/nscd/socket"
```

Comment:
/var/run? Isn't /run the new default?

Reply:
Something like `RUNDIR` should be defined in config.h

Reply:
Yes, but that is an migration fix until all has moved to /run

Reply:
I may be confused, but it seems to be the right path.

https://elixir.bootlin.com/glibc/latest/source/nscd/nscd-client.h#L41